sites/advertiser/private-aggregation/k-freq-measurement.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+async function injectAd() {
+  // Load the Shared Storage worklet 
+  await window.sharedStorage.worklet.addModule('k-freq-measurement-worklet.js');
+
+  // Set the initial impression count
+  window.sharedStorage.set('impression-count', 0, {
```

Comment:
Does this reset the impression count to zero on each viewing?

Reply:
No. The `ignoreIfPresent: true` option skips setting the value on subsequent visits.